uportal-war/src/main/java/org/jasig/portal/portlets/tenantmanager/TenantManagerController.java

```diff
@@ -78,13 +121,55 @@ public void doAddTenant(ActionRequest req, @RequestParam("name") String name,
             }
         }
 
-        tenantService.createTenant(name, fname, attributes);
+        List<TenantOperationResponse> responses = new ArrayList<>();
+        tenantService.createTenant(name, fname, attributes, responses);
+
+        // Need to store some items to share with user in the report;  would be
+        // handy to have support for javax.portlet.actionScopedRequestAttributes
+        session.setAttribute(OPERATION_NAME_CODE, "tenant.manager.add");
+        session.setAttribute(OPERATIONS_LINTENER_RESPONSES, responses);
+
+        // Send the user to the report screen
+        res.setRenderParameter("action", "showReport");
```

Comment:
DRY Principal:  refactor session manipulation and FWD to showReport to a private method.